### ISSUE TITLE
Axe core step definition has incorrect run options config

### ISSUE THREAD

**Issue by dylankelly** [dpc-sdp/ripple#1029]
**Describe the bug**

**To Reproduce**
Steps to reproduce the behaviour:
1. Add a test which uses the step definiton "it has no detectable a11y violations on load"

**Expected behaviour**

Test uses Axe core to check page for a11y issues

**Actual behaviour**

Cypress throws error "TypeError: axe.run arguments are invalid"

**Screenshots**

**Versions**
- Ripple: 

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**
